aws-cli/bash-linux/dynamodb/scenario_getting_started_movies.sh

```diff
@@ -0,0 +1,614 @@
+
+  get_input_result="$response"
+
+  return 0
+}
+
+###############################################################################
+# function get_float_input
+#
+# This function requests a non-negative float answer from the user.
+#
+# Parameters:
+#       $1 - The prompt.
+#       $2 - Optional inclusive low.
+#       #3 - Optional inclusive high.
+#
+# Returns:
+#       The input,
+#    And
+#       0 - If successful.
+#       1 - If an error occurred.
+###############################################################################
+function get_float_input() {
+  if [ -z "$1" ]; then
+    echo "Internal error get_float_input"
+    return 1
+  fi
+
+  local low=$2
+  local high=$3
+
+  local index=0
+  local response=""
+  local input_valid=false
+  while [[ $input_valid == false ]]; do
+    index=$((index + 1))
+    if [[ index -gt 10 ]]; then
+      return 1
+    fi
+
+    echo -n "$1"
+    get_input
+
+    # Remove leading integers.
+    # shellcheck disable=SC2001
+    response=$(echo "$get_input_result" | sed "s/[[:digit:]]*//")
+
+    if [[ -n "$response" ]]; then # Remove decimal if present.
+      # shellcheck disable=SC2001
+      response=$(echo "$response" | sed "s/^\.[[:digit:]]*//")
+    fi
+
+    # Non-empty string indicates invalid input.
+    if [[ -n "$response" ]]; then
+      echo -e "Please enter a floating point number."
+      continue
+    fi
+
+    response="$get_input_result"
+
+    if [[ -z $low ]] || [[ -z $high ]]; then
+      break
+    fi
+
+    if (($(echo "$response < $low" | bc -l))); then
+      echo -e "Response must be greater than or equal to $low."
+      continue
+    fi
+
+    if (($(echo "$response > $high" | bc -l))); then
+      echo -e "Response must be less than or equal to $high."
+      continue
+    fi
+
+    input_valid=true
+  done
+
+  get_input_result="$response"
+  return 0
+}
+
+###############################################################################
+# function echo_repeat
+#
+# This function prints a string 'n' times to stdout.
+#
+# Parameters:
+#       $1 - The string.
+#       $2 - Number of times to print the string.
+#
+# Outputs:
+#   String 'n' times to stdout.
+#
+# Returns:
+#       0
+###############################################################################
+function echo_repeat() {
+  local end=$2 i
+  for ((i = 0; i < end; i++)); do
+    echo -n "$1"
+  done
+  echo
+}
+
+# snippet-start:[aws-cli.bash-linux.dynamodb.scenario_getting_started_movies]
+###############################################################################
+# function dynamodb_getting_started_movies
+#
+# Scenario to create an Amazon DynamoDB table and perform a series of operations on the table.
+#
+# Returns:
+#       0 - If successful.
+#       1 - If an error occurred.
+###############################################################################
+function dynamodb_getting_started_movies() {
+  source ./dynamodb_operations.sh
+
+  key_schema_json_file="dynamodb_key_schema.json"
```

Comment:
Does the example provide samples for these files?

Reply:
These files are written to, passed as parameters, and then deleted on cleanup. There should not be included samples.